### PR TITLE
Github pages to replace wiki

### PR DESCRIPTION
**Pull request by ntotten**
Adds github pages site to replace wiki and other documentation for VS Code.

image

### CODE REVIEW

File: contributing/commit-guidelines.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Introduction
+# Committ Guidelines
```
Comment (vazexqi) [inline contributing/commit-guidelines.md]: Typo. `Committ` -> `Commit`.

File: contributing/coding-guidelines.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# Codeing Guidelines
```
Comment (vazexqi) [inline contributing/coding-guidelines.md]: Typo. `Codeing` -> `Coding`.

File: contributing/dependencies.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Dependencies
```
Comment (vazexqi) [inline contributing/dependencies.md]: I'm not sure if this file is necessary. It was something was added by Gunnar a while back. Not sure if this is actually used. I am also not sure if this is up to date.

File: docs/_data/sidebar.yml [hunk 4]
```diff
@@ -0,0 +1,29 @@
+- title: Getting Started
+  children:
```
Comment (vazexqi) [inline docs/_data/sidebar.yml]: I think you need to remove this sample `children` section.

File: docs/_includes/footer.html [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+            >Responsible Disclosure</a
+          >
+        </li>
+        <li class="slds-item">
+          <a href="https://trust.salesforce.com/" target="_blank">Trust</a>
+        </li>
+      </ul>
+    </nav>
+    <p>
+      © Copyright 2015-present Salesforce.com, inc.&nbsp;<a
```
Comment (vazexqi) [inline docs/_includes/footer.html]: Why is the copyright year for this 2015?

Comment (ntotten) [reply docs/_includes/footer.html]: Changed to use current year.

File: docs/_data/sidebar.yml (hunk 4 above)
Comment (ntotten) [reply docs/_data/sidebar.yml]: WIP, it has real pages now.

File: contributing/dependencies.md (hunk 3 above)
Comment (ntotten) [reply contributing/dependencies.md]: Deleted.

File: contributing/coding-guidelines.md (hunk 2 above)
Comment (ntotten) [reply contributing/coding-guidelines.md]: fixed

File: contributing/commit-guidelines.md (hunk 1 above)
Comment (ntotten) [reply contributing/commit-guidelines.md]: fixed

**Comment by ntotten** [comment]
@vazexqi It does now. I wasn't done yet. ;)

**Comment by ruthemmanuelle** [comment]
This is great, @ntotten . I'm working on a pull request to your `docs` branch that will contain various edits. Some more general things I've noticed thus far on http://localhost:4000 (after running `bundle exec jekyll serve`):

- Headings preceded by more than two pound signs (level-three headings or lower) are shown as plain text.
- `***` or `---` doesn't produce a horizontal line unless used directly above or below a heading. When not adjacent to a heading, `---` changes the sizing of the text it's adjacent to and `***` shows as asterisks.
- One of your screenshots (shared via Slack) showed a search bar, but I'm not seeing one.

I'll submit more feedback as soon as I can.

**Comment by ntotten** [comment]
Thanks @ruthemmanuelle. 

1. I'll make a fix to the headings, good catch.
2. I am working on search, we need an account with Alogila first. I plan on adding it soon, but I didn't want to block on it.

**Comment by ruthemmanuelle** [comment]
@ntotten - I noticed yesterday that level-one headings (with one pound sign) are also shown as plain text. (I'd mentioned this issue previously re: level-three headings.) It seems like the only pound-sign-preceded headings that show up as headings are `##` headings.

**Comment by ruthemmanuelle** [comment]
Hi @ntotten - Another issue I noticed: Some pages don't have any headings other than a title, so they show "Contents" on the right with nothing under it. Do you know whether it's possible to modify `docs/_layouts/default.html` or `docs/_includes/sticky-nav.html` to suppress the site menu if `contents-list` is empty?

**Comment by ruthemmanuelle** [comment]
Hi @ntotten - One more thing that would be nice to change: The contents list (on the right side of each topic) uses the same spacing when section names wrap that it uses between section names. It would be nice if it added extra space between the links, like the sidebar on the left does.

**Comment by ntotten** [comment]
@ruthemmanuelle I wasn't able to reproduce the issue with `---`

image

**Comment by ntotten** [comment]
@ruthemmanuelle Your suggested changes have been made. Feel free to review when you have a chance. Thanks!

**Comment by ntotten** [comment]
Added search as well:

image

**Pull request merged**